crates/bevy_reflect/src/serde/de.rs

```diff
@@ -273,7 +274,7 @@ impl<'a, 'de> Visitor<'de> for UntypedReflectDeserializerVisitor<'a> {
         A: MapAccess<'de>,
     {
         let type_name = map
-            .next_key::<String>()?
+            .next_key::<Cow<'static, str>>()?
```

Comment:
Can we try to constrain the borrow here to `'de` instead of `'static`?

Reply:
Good call.